apps/rush-lib/src/schemas/rush.schema.json

```diff
@@ -93,6 +94,22 @@
             "fewer-dependencies",
             "fast"
           ]
+        },
+        "environmentVariables": {
```

Comment:
There is a convention for "extending" objects in JSON schemata. Let's do that instead of duplicating this.